<launch>

    <include file="$(find-pkg-share roboclaw_ros2)/launch/roboclaw_single.launch.xml"/>

    <node pkg="roboclaw_ros2" exec="diffdrive_node" name="diffdrive" output="screen">
        <param name="roboclaws" value="1"/>
        <param name="steps_per_meter" value="4100.0"/> <!-- 4288.382 -->
        <param name="base_width" value="0.48"/>
        <param name="swap_motors" value="false"/>
        <param name="invert_motor_1" value="false"/>
        <param name="invert_motor_2" value="false"/>

        <param name="var_pos_x" value="0.01"/>
        <param name="var_pos_y" value="0.01"/>
        <param name="var_theta_z" value="0.01"/>
        <param name="odom_tf_name" value="odom"/>  
        <param name="base_tf_name" value="rover/base_footprint"/>
        <param name="use_runge_kutta" value="true"/>

    </node>

</launch>
